src/app.js

```diff
@@ -24,13 +25,15 @@ import './polyfills';
 import configureStore from './configureStore';
 
 const environment = process.env.NODE_ENV;
+const dsn = configuration.sentry && configuration.sentry.dsn;
 const release = process.env.GIT_COMMIT;
-
-Sentry.init({
-  environment,
-  dsn: 'https://3de59e3a93034a348089131aa565bdf4@sentry.data.amsterdam.nl/27',
-  release,
-});
+if (dsn) {
```

Comment:
When this condition doesn't meet, then the Sentry module also shouldn't be imported. That also goes for Matomo. We could go for a `noop` module replacement in that case. That should then be applied to the Webpack settings.

Reply:
Also, in multiple places in the code the `captureException` method of the Sentry instance is called, so that shouldn't break either.

Reply:
While not the cleanest solution, it doesn't pose a problem to import Sentry and call `captureException` when it's not initialized.

Reply:
The `MatomoTracker` is only called in `app.js` within that if-statement.